test/test_testing.py

```diff
@@ -3002,6 +3002,12 @@ def test_makedir_fails_if_already_exists(self):
             self.fs.create_dir('/etc')
         self.assertEqual(cm.exception.args[0], '/etc')
 
+    def test_makedir_succeeds_if_already_exists_when_make_parents_true(self):
```

Comment:
Is this test part of the mix in that runs against the real Pebble backend when the right flags are set? If you've tested it yourself to confirm, I'm ok with the change, but if we can get a test to ensure the testing infrastructure matches the live backend even better.

Reply:
Just tested it locally with a real pebble -- all good for this change :) There is one failing test ref. permissions which we are already tracking, I believe.

Reply:
(And yes, it's part of the mixin :))